go/oasis-test-runner/oasis/byzantine.go

```diff
@@ -91,6 +91,10 @@ func (net *Network) NewByzantine(cfg *ByzantineCfg) (*Byzantine, error) {
 		return nil, err
 	}
 
+	sp := ""
+	if net.cfg.UseCustomGrpcSocketPath {
+		sp = net.getTempSocketPath()
```

Comment:
Since this only requires `net`, you could maybe move this to `net.startOasisNode` and avoid repeating the same thing all over?